steps/cadence-genus-genlib/scripts/read_design.tcl

```diff
@@ -0,0 +1,34 @@
+set genus_design_name              $::env(design_name)
+set genus_gl_netlist               [glob -nocomplain inputs/*.vcs.v]
```

Comment:
These globs are not good! Please instead use [lsort [glob...]] ... see recently filed garnet ref 
https://github.com/StanfordAHA/garnet-refs/issues/43

Reply:
Note there are more globs nearby that need fixing, i.e. lines 3-12 in the same file `read_design.tcl`...for some reason my comment did not include all of them...

Reply:
Thanks Steve, fixed.